asf_lifetime_cost_model/pipeline/cost_computations.py

```diff
@@ -0,0 +1,236 @@
+"""Cost computations.
+
+Functions to handle cost computations for the ASF lifetime cost model including:
+- Total maintenance costs
+- Upfront installation costs
+- Running costs
+- Total lifetime costs computation
+as a total and as a breakdown per year in the lifetime of the heating system.
+"""
+
+# package imports
+from typing import Union
+
+# local imports
+from asf_lifetime_cost_model import config
+from asf_lifetime_cost_model.getters import data_getters
+
+
+def compute_total_maintenance_cost(annual_maintenance_costs: float, life_span: int) -> float:
+    """Computes the total maintenance cost for technology given the annual maintenance cost and the expected lifetime.
+
+    Args:
+        annual_maintenance_costs (float): The annual maintenance cost of the heating system.
+        life_span (int): Number of years the heating system is assumed to be operational.
+
+    Returns:
+        float: The total maintenance costs over the specified number of years.
+    """
+    return annual_maintenance_costs * life_span
+
+
+def creates_cost_reduction_data(
```

Comment:
```suggestion
def create_cost_reduction_data(
```

Not sure if using 'creates' instead of 'create' was an intentional naming convention style - but think it's clearer to stay consistent across all function names :)

Reply:
Yep - it's a typo, thank you! :)